doc/conf.py

```diff
@@ -222,22 +178,14 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
 
-# Sometimes the savefig directory doesn't exist and needs to be created
-# https://github.com/ipython/ipython/issues/8733
-# becomes obsolete when ipython 5.2 is out
-ipython_savefig_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)),
-                                   '_build','html','_static')
-if not os.path.exists(ipython_savefig_dir):
-    os.makedirs(ipython_savefig_dir)
-
```

Comment:
We pin all our packages in the doc build (https://github.com/pydata/xarray/blob/master/doc/environment.yml) and currently still use ipython 5.1. I'd rather leave this workaround here for now.

(unpinning the package versions is a painful trial and error process which I can only do on the repo itself -- this is the purpose of the fix-docs branch)